core/src/test/scala/tech/beshu/ror/unit/acl/domain/ActionTest.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.unit.acl.domain
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import tech.beshu.ror.accesscontrol.domain.Action
+
+class ActionTest extends AnyWordSpec with Matchers {
```

Comment:
OK, but IMO we need one more test. Let's provide a core integration test (the one that extends BaseYamlLoadedAccessControlTest) and check if `Action` rule with the old ROR action name will be matched now